components/resources/brave_components_strings.grd

```diff
@@ -533,6 +533,9 @@
         Pending until <ph name="DATE">$1</ph>
       </message>
 
+      <message name="IDS_REWARDS_BROWSER_CANNOT_RECEIVE_ADS" desc="">Oops! Your browser cannot receive Brave Private Ads.</message>
+      <message name="IDS_REWARDS_BROWSER_NEED_UPDATE_TO_SEE_ADS" desc="">Your browser needs to be updated to continue seeing ads.</message>
```

Comment:
I think I mistyped my naming suggestion 🤦  sorry about that! It should probably be `IDS_REWARDS_BROWSER_NEEDS_UPDATE_TO_SEE_ADS` to match the other names.

Reply:
fixed